### PR TITLE
[Stream] Prometheus metrics

### PR DESCRIPTION
**Pull request by JackyWYX**
## Issue

https://github.com/harmony-one/harmony/issues/3555

Added a lot of prometheus metrics to stream protocol. 

## Test

Tested locally. Also deployed to testnet. 

Following is the result metrics pulled from ${IP}:9900/metrics:
https://gist.github.com/JackyWYX/0997fa98fe8958ee6b3a58f0a5a21330

### CODE REVIEW

**Comment by rlan35** [comment]
generally looks good. Waiting for @LeoHChen 's final approval.

**Review by LeoHChen — APPROVED** [review]
LGTM. Can you add a test result log into the PR, so that we can see the metrics.

**Comment by JackyWYX** [comment]
Here is the example metrics: 

https://gist.github.com/JackyWYX/0997fa98fe8958ee6b3a58f0a5a21330

**Pull request merged**
